frontend/src/v5/ui/components/shared/modals/modals.styles.ts

```diff
@@ -15,24 +15,20 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 import { DialogContent } from '@mui/material';
 import { Typography } from '@controls/typography';
 
-export const Container = styled.div`
+export const ModalContainer = styled.div`
 	display: flex;
 	flex-direction: column;
 	align-items: center;
 	min-width: 633px;
-	min-height: 295px;
-	padding-top: 43px;
+	padding: 43px 0;
```

Comment:
Yes. Before it was always adding this space. But instead of padding the border we had logic telling the last component to have a bottom margin. Or in the case of the Alert modal I think it just relied on the dialog modal's min-height.
Although I think the bottom padding should be '25px' as it was before, so I will change that